### ISSUE TITLE
Download Button needs discernible text

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7188]
**CKAN version**
https://demo.ckan.org/dataset/sample-dataset-1/resource/e687245d-7835-44b0-8ed3-0827de123895

**Describe the bug**
Title: WCAG 4.1.2: Ensures buttons have discernible text (.dropdown-toggle)
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Target application: Sample Dataset - sample-linked.csv - CKAN Demo - https://demo.ckan.org/dataset/sample-dataset-1/resource/e687245d-7835-44b0-8ed3-0827de123895

Element path: .dropdown-toggle

Snippet: <button class="btn btn-primary dropdown-toggle" data-toggle="dropdown"> <span class="caret"></span> </button>

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 107.0.1418.35

====

This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
